marconi/test/Integration.hs

```diff
@@ -316,7 +316,11 @@ testIndex = H.integration . HE.runFinallies . workspace "chairman" $ \tempAbsBas
     let
       queryLoop n = do
         H.threadDelay 250_000 -- wait 250ms before querying
-        txCbors <- liftIO $ ScriptTx.query indexer (ScriptTx.ScriptAddress plutusScriptHash) []
+        -- With the new indexer, this should pass right away (by finding the in-memory
```

Comment:
If the loop isn't required anymore, then just remove it!

Reply:
It's not that it is not required. It is required if we want to check the data that was stored in the database. I don't think it's a bad idea to do it like this. @koslambrou is also interested in testing that the SQL queries work properly, so I think this is a good thing to have.